src/components/Notes.tsx

```
@@ -54,7 +57,19 @@ const Notes: React.FC<NotesProps> = ({ notesData }) => {
           </p>
           {renderTags(selectedNote.tags)}
           <div className="prose dark:prose-invert text-neutral-700 dark:text-neutral-300 mt-4">
-            <p>{selectedNote.content}</p>
+            <ReactMarkdown
+              children={selectedNote.content}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)